the-rules/data/no-rules.json5

```diff
@@ -7,5 +7,13 @@
   // You may want to know why someone needs branches if there are no rules, well its a story about time
   // if I had more time I would explain it, but I'm still working on the rules of math, which are not so much rules as ...
   // sorry that may have been a rant there, keep going, and just define your own truth.
+  ruleOne: {
+    active: true,
+    parameters : [
+     "many different string statements allowed",
```

Comment:
we would need to prompt the user if we want to assert what is allowed and what is not.